### ISSUE TITLE
Enhancement: Add Media Devices 

### ISSUE THREAD

**Issue by auspokey** [t0bst4r/home-assistant-matter-hub#57]
Would be nice to have the ability to have Televisions or media devices in the bridge

**Comment by t0bst4r** [comment]
You are right, I forgot about those. BUT no vendor (apple, amazon, google, Samsung) support media players via matter. So I’ll map them to simple on-off-switches.

**Comment by ThatBlockyPenguin** [comment]
> So I’ll map them to simple on-off-switches.

I personally would rather have a configurable option for this. I would prefer to have my devices show up as what they actually are (TVs, speakers, whatever), and wait for vendors to add functionality. I know obviously most people wouldn't want this though, which is why I'm voting for a config toggle.

**Comment by t0bst4r** [comment]
yes i understand that, but starting with the on-off switch will be easier for me. I'd need to dive deep into how media players work with matter - i already know that this is a lot harder than other device types.

one addition which speaks (at least) for the configuration option: if you are adding an unsupported device to the bridge, at least alexa won't connect the whole bridge. so one will not able to control any of the other devices.
